Pipfile

```
@@ -81,13 +77,14 @@ libsass = "*"
 django-formtools = "==2.2"
 django-debug-toolbar = "*"
 django-filter = "*"
-click = "==8.0.4"
-flask = "==2.0.2"
+click = "*"
+flask = "==2.3.2"
+certifi = "*"
 
 [scripts]
 collectstatic = "python manage.py collectstatic -c --noinput"
 createcachetable = "python manage.py createcachetable"
-pytest = "pytest"
 coveralls = "coveralls"
 migratedb = "python manage.py migrate --noinput"
-thumbs_sam_build = "bash -c 'cd thumbs && sam build'"
+thumbs_sam_build = "bash -c 'DJANGO_SETTINGS_MODULE=electionleaflets.settings.testing python manage.py test'"
```

This should still be

```suggestion
thumbs_sam_build = "bash -c 'cd thumbs && sam build'"
```